src/main/java/gregtech/api/capability/impl/FuelRecipeLogic.java

```diff
@@ -207,21 +224,88 @@ public long getMaxVoltage() {
         return maxVoltage;
     }
 
+    protected double calculateFuelConsumptionMultiplier() {
+        return 1.0;
+    }
+
     protected int calculateFuelAmount(FuelRecipe currentRecipe) {
-        return currentRecipe.getRecipeFluid().amount * getVoltageMultiplier(getMaxVoltage(), currentRecipe.getMinVoltage());
+        return (int) (currentRecipe.getRecipeFluid().amount * getVoltageMultiplier(getMaxVoltage(), currentRecipe.getMinVoltage()) * calculateFuelConsumptionMultiplier());
+    }
+
+    public int getFuelAmount() {
```

Comment:
Same issue like with getOutputVoltage: getters should return cached values, not calculate new ones